solarbextrapolation/potential_field_extrapolator.py

```diff
@@ -149,13 +84,36 @@ def _extrapolate_phi(self, debug=False):
         
         # Parameters
         arr_boundary = self.map_boundary_data.data
-        d = 1.0        
-    
+        enable_numba = kwargs.get('enable_numba', True)
+        
+
+        if enable_numba:
+            try:
+                import numba
+                from potential_field_extrapolator_numba import *
+                return phi_extrapolation_numba(arr_boundary, self.shape, self.Dx.value, self.Dy.value, self.Dz.value)
```

Comment:
rather than repeating the call twice I would do this:

```python
if enable_numba:
    try:
        import numba
        from potential_field_extrapolator_numba import phi_extrapolation_numba as phi_extrapolation
    except ImportError:
        from potential_field_extrapolator_python import phi_extrapolation_python as phi_extrapolation
return phi_extrapolation(arr_boundary, self.shape, self.Dx.value, self.Dy.value, self.Dz.value)
```